jcl/src/java.base/share/classes/jdk/internal/misc/Unsafe.java

```diff
@@ -45,52 +45,52 @@ private Unsafe() {}
 	/**
 	 * Represents an invalid field offset value.
 	 */
-	public static final int INVALID_FIELD_OFFSET;
+	public static final long INVALID_FIELD_OFFSET;
```

Comment:
These changes must be version-specific, otherwise it breaks earlier versions; e.g. in Java 24:
```
src/java.base/share/classes/jdk/internal/foreign/Utils.java:304: error: incompatible types: possible lossy conversion from long to int
                new BaseAndScale(Unsafe.ARRAY_BYTE_BASE_OFFSET, Unsafe.ARRAY_BYTE_INDEX_SCALE);
                                       ^
```

Reply:
Fixed.